test/link-style-consistent.md

```diff
@@ -0,0 +1,14 @@
+# Consistent Link Style
+
+[url {MD054}](http://example.com)
```

Comment:
Might be nice to have "Text EXAMPLE text" to help make sure fix column isn't 0-based.